xds/internal/xdsclient/xdsresource/filter_chain.go

```diff
@@ -21,15 +21,17 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"unsafe"
+
+	"google.golang.org/grpc/internal/resolver"
+	"google.golang.org/grpc/xds/internal/httpfilter"
+	"google.golang.org/grpc/xds/internal/xdsclient/xdsresource/version"
 
 	v3listenerpb "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
 	v3httppb "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 	v3tlspb "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
```

Comment:
These type lines need to be with the other ones on top as well.

Reply:
Moved alongside section of google.golang.org/grpc. I thought these logically belong in the proto bucket.